apps/test/unit/lib/kits/maker/boards/microBit/MBFirmataUpdaterTest.js

```diff
@@ -0,0 +1,87 @@
+import {expect} from '../../../../../../util/reconfiguredChai';
+import {
+  MICROBIT_V1,
+  MICROBIT_V2,
+  MICROBIT_FIRMATA_V1_URL,
+  MICROBIT_FIRMATA_V2_URL
+} from '@cdo/apps/lib/kits/maker/boards/microBit/MicroBitConstants';
+import MBFirmataUpdater from '@cdo/apps/lib/kits/maker/boards/microBit/MBFirmataUpdater';
+import microBitReducer, {
+  setMicroBitFirmataUpdatePercent
+} from '@cdo/apps/lib/kits/maker/microBitRedux';
+import {
+  getStore,
+  registerReducers,
+  stubRedux,
+  restoreRedux
+} from '@cdo/apps/redux';
+
+describe('MBFirmataUpdater', () => {
+  describe('detectMicroBitVersion function', () => {
+    it('returns correct version for micro:bit v1', () => {
+      const mbFirmataUpdater = new MBFirmataUpdater();
```

Comment:
nit: You could just declare one `mbFirmataUpdater` at the top of the `describe('detectMicroBitVersion function', () => {})` block.